syntax = "proto3";
package Protocol;

import "Enum.proto";
import "Struct.proto";

message C2P_RequestLogin
{
	string id = 1;
	string pw = 2;
}

message P2C_ResultLogin
{
	uint32 result = 1;
}

message C2P_ReportMove
{
	int64 sessionKey = 1;
	UserData user = 2;
}

message P2C_ReportMove
{
	int64 sessionKey = 1;
	UserData user = 2;
}

message C2P_RequestCollison
{
	int64 sessionKey = 1;
	UserData src = 2;
	repeated MonsterData descDatas = 4;
}

message P2C_ResultCollision
{
	uint32 result = 1;
	UserData src = 2;
	repeated MonsterData descDatass = 5;
}

message P2C_ReportUpdateMonsters
{
	repeated MonsterData datas = 5;
}